Resources/doc/filters.md

```diff
@@ -0,0 +1,98 @@
+# Filters
+
+The bundle provides a generic system to apply filters on collections. It ships with built-in Doctrine ORM support
+and can be extended to fit your specific needs.
+
+By default, all filters are disabled. They must be enabled manually.
+
+When a filter is enabled, it is automatically documented as a `hydra:search` property in collection returns. It also automatically
+appears in the NelmioApiDoc documentation if this bundle is installed and enabled.
+
+## Adding Doctrine ORM filters
```

Comment:
You't right, we should think about that. I don't like the "where" term. What about "SearchFilter"?